library/pk.c

```diff
@@ -572,9 +572,10 @@ int mbedtls_pk_wrap_as_opaque( mbedtls_pk_context *pk,
         return( ret );
 
     curve_id = mbedtls_ecp_curve_info_from_grp_id( ec->grp.id )->tls_id;
+    key_type = PSA_KEY_TYPE_ECC_KEYPAIR(curve_id);
```

Comment:
I think we should have a dummy helper function for the conversion from TLS identifiers to PSA identifiers, even if - currently - it's the identity - see e.g. https://github.com/ARMmbed/mbedtls/pull/2338.